extensions/dicom-microscopy/src/DicomMicroscopyViewport.tsx

```diff
@@ -172,15 +172,15 @@ function DicomMicroscopyViewport({
       microscopyService.clearAnnotations();
 
       let smDisplaySet = displaySet;
-      if (displaySet.Modality === 'SR') {
+      if (displaySet.Modality === 'SR' || displaySet.Modality === 'ANN') {
```

Comment:
Can you capture this in a function/constant so that it is consistent and that you re-use the detection code?  What I like doing for it is to set an attribute on the display set that indicates that this is an overlay for slide microscopy.  That way it is the display set that decides this use case.

Reply:
This then becomes if (displaySet.isWSIOverlay) ...

Reply:
I think the best approach is to use the CS3D microscopy viewport, right? This is just the basic ann support for connecathon purposes.